crates/beerus-cli/tests/cli.rs

```diff
@@ -3609,44 +3610,47 @@ mod test {
             Box::new(starknet_lightclient),
         );
 
-        let program = vec![];
+        let sierra_program = vec![];
         let constructor = vec![ContractEntryPoint {
-            offset: 10,
+            function_idx: 10,
             selector: FieldElement::from_str("0").unwrap(),
         }];
 
         let external = vec![ContractEntryPoint {
-            offset: 10,
+            function_idx: 10,
             selector: FieldElement::from_str("0").unwrap(),
         }];
 
         let l1_handler = vec![ContractEntryPoint {
-            offset: 10,
+            function_idx: 10,
             selector: FieldElement::from_str("0").unwrap(),
         }];
         let entry_points_by_type = EntryPointsByType {
             constructor,
             external,
             l1_handler,
         };
-        let abi = None;
+        let abi = Default::default();
+        let contract_class_version = Default::default();
 
-        let contract_class: ContractClass = ContractClass {
-            program,
+        let contract_class = SierraContractClass {
```

Comment:
These needed to be updated as part of updating AddDeclareTransaction to latest version.